### DISCUSSION TITLE
Tutorial on Flutter Web & Mobile

### DISCUSSION THREAD

**Discussion by ajay01994** [medusajs/medusa#2050] [Q&A] [answered]
Hi Guys , thanks for releasing Medusa among the  open source community ,  Really it's a great js e-com project , I have gone through tutorial using Next.js &  Gatsby which are very easy to understand , i would request the community to  share an extensive tutorial on making an e-com store using Flutter web and mobile ,it will help a lot to all the developers ..

Thanks !

**Comment by MedusaNick** [comment, marked as answer]
One of our community members made a tutorial on this recently. Might be worth checking out: https://dev.to/iamvictorsam/creating-an-ecommerce-app-with-flutter-and-medusa-4i52?
